package/utils/uencrypt/Makefile

```
@@ -23,15 +23,16 @@ define Package/uencrypt
   SECTION:=utils
   CATEGORY:=Base system
   TITLE:=Decryption utility for Arcadyan WG4xx223
-  DEPENDS:=@TARGET_ramips_mt7621 +UENCRYPT_WOLFSSL:libwolfssl +UENCRYPT_OPENSSL:libopenssl
+  DEPENDS:=@(TARGET_ramips_mt7621||TARGET_ath79_generic_DEVICE_tplink_deco-s4-v2) +UENCRYPT_WOLFSSL:libwolfssl +UENCRYPT_OPENSSL:libopenssl
```

```suggestion
  DEPENDS:=+UENCRYPT_WOLFSSL:libwolfssl +UENCRYPT_OPENSSL:libopenssl
```
Why not let anyone install it?  I think it has become useful enough that we should lift this restriction.

---

agreed.

i already PACKAGES+= it to make sure it appears in a deco-targeted build (seperate pull request)
assume you do similar with ramips_mt7621 target?